docs/api/clients/typescript.md

```diff
@@ -33,8 +33,7 @@ import { ShapeStream } from '@electric-sql/next'
 
 // Passes subscribers rows as they're inserted, updated, or deleted
 const stream = new ShapeStream({
-  baseUrl: `${BASE_URL}`,
-  shape: { table: `foo` }
+  url: `${BASE_URL}/v1/shape/foo`,
```

Comment:
Is it meant to use `BASE_URL` here?
It's not really defined in the example, so perhaps just a localhost url is more clear?

Reply:
yeah true, will fix